src/orquestra/sdk/_base/_workflow.py

```diff
@@ -314,13 +315,15 @@ def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> WorkflowDef[_R]:
                 isinstance(called_fn.function, TaskDef)
                 or called_fn.function in allowed_function_calls
             ):
-                warnings.warn_explicit(
-                    f'"{called_fn.name}" is not a task. Did you mean to decorate '
-                    "it with @task?",
-                    NotATaskWarning,
-                    called_fn.source_file or "<unknown>",
-                    called_fn.line_no or 0,
-                )
+                non_task_functions.append(called_fn.name)
+
+        if non_task_functions:
+            warnings.warn(
+                f'Functions: "{", ".join(non_task_functions)}" called in the workflow '
+                f"are not a tasks. Did you mean to decorate it with @task?",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Let's ignore the singular case and assume there's more than one.
```suggestion
                f"are not a tasks. Did you mean to decorate them with @task?",
```

Reply:
additional tweak:
```suggestion
                f"are not tasks. Did you mean to decorate them with @task?",
```